.storybook/components/ColorSwatches/ColorSwatches.module.css.d.ts

```diff
@@ -0,0 +1,6 @@
+declare const styles: {
```

Comment:
I see this change in a lot (all) of the css.d.ts files in the repo, is this change a requirement for this PR?

Reply:
Nope it's a side-effect of a merge with master. I posted about it internally, and got a response. I just need to delete them manually from the repo and re-push, they should be gone before the merge.